src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -2202,6 +2203,94 @@ test_slow_server_pooled (void)
    mock_server_destroy (primary);
    checks_cleanup (&checks);
 }
+
+static void
+_test_ismaster_versioned_api (bool pooled)
+{
+   mock_server_t *server;
+   mongoc_uri_t *uri;
+   mongoc_client_pool_t *pool;
+   mongoc_client_t *client;
+   char *ismaster;
+   future_t *future;
+   request_t *request;
+   bson_error_t error;
+   mongoc_server_api_version_t version;
+   mongoc_server_api_t *api;
+
+   server = mock_server_new ();
+   mock_server_run (server);
+   uri = mongoc_uri_copy (mock_server_get_uri (server));
+
+   mongoc_server_api_version_from_string ("1", &version);
+   api = mongoc_server_api_new (version);
+
+   if (pooled) {
+      pool = mongoc_client_pool_new (uri);
+      ASSERT_OR_PRINT (mongoc_client_pool_set_server_api (pool, api, &error),
+                       error);
+
+      client = mongoc_client_pool_pop (pool);
+   } else {
+      client = mongoc_client_new_from_uri (uri);
+      ASSERT_OR_PRINT (mongoc_client_set_server_api (client, api, &error),
+                       error);
+   }
+
+   ismaster = bson_strdup_printf ("{'ok': 1,"
+                                  " 'ismaster': true,"
+                                  " 'setName': 'rs',"
+                                  "  'minWireVersion': 2,"
```

Comment:
super nit, weird spacing

Reply:
Copypasta strikes again. I've aligned the indentation across all occurrences in the file.